modules/Bio/EnsEMBL/Compara/PipeConfig/ProteinTrees_conf.pm

```diff
@@ -3222,9 +3222,7 @@ sub core_pipeline_analyses {
                 'exclude_list'  => 1,
                 'output_file'   => '#dump_dir#/snapshot_3_after_tree_building.sql.gz',
             },
-            -flow_into  => {
-                1 => WHEN('#do_gene_qc#' => 'get_species_set'),
```

Comment:
With this change I would suggest to downgrade `do_gene_qc` to standard parameter instead of pipeline wide parameter and pass it to `HomologyGroupingFactory` "manually".